### ISSUE TITLE
RFC: Informing the user to increase analysis quality

### ISSUE THREAD

**Issue by cchristelis** [inasafe/inasafe#1234]
# Problem

The quality of the analysis results are directly related to the quality of the input data. The user should be informed about how their choice of input data (hazard and exposure) propagates through to the analysis results. 
The user should be informed about all other aspects which affect the analysis quality and results.
The user should be presented with appropriately precise results.
# Rationale

This RFC attempts to be a collection point for all matters related to enhancing the analysis quality that will directly affect the ability of uses to make decisions. Aspects of interest may include the training material, IF, metadata, reports, etc. 
# Current State

Draft
# Proposers

@Charlotte-Morgan 
@cchristelis
# Detail
## Scale

One way to think about the issue  is by scale. The scale of the data could be categorized into classes such as:
1. Village
2. District
3. Province
4. National

The classification of all data can be related to the capture scale which can be obtained from metadata if available. Users should be careful with the classification of _Raster_ data, a small cell size does not mean a large capture scale. 

The details of this scale classification will be followed up in parallel with the metadata issues.
And can be discussed with end users to ensure the scale of potential InaSAFE application is reflected in the proposed scale classes above. 
## Overestimation

Since InaSAFE is used for disaster preparedness and disaster response, it is appropriate to overestimate rather than underestimate impacts. For example, if any part of a building is flooded, the whole building should be considered flooded. This is relevant to both the flood and tsunami impact functions.

The 'flooded' depth report should return 3 classes and not the current binary flooded and not flooded based on the user determined threshold. At present the not inundated buildings are used in the action list to imply that the building is open for business and potentially available to be used as an evacuation center. In preference, the impact function, will return 3 classes (when the user specifies a single threshold):
1. flooded (for cases that are above the set threshold)
1.  wet (for cases below the threshold, but within water) 
1. dry for buildings (no water).
## Extents

The analysis extent should be considered in light of the hazard and exposure data scales. The user should be encouraged to undertake the analysis on an extent suited to the data scale; or find data at an appropriate scale to their proposed analysis extent.
## Appropriate Accuracy

The user should get an accuracy that is suited to the analysis. (hmm - dont get this bit)
# Proposed Solution

Once a resolution is determined for any of the aspects discussed here, it should be noted and followed up in another issue.  (Please add the actual issue number below.)
## Scale
1. _Training Material_ should inform the user about how the scale of the input hazard and exposure data, when combined together will affect the scale at which decisions can be made. The following graphic may assist showing appropriate regions for the given hazard and exposure:
   ![screenshot from 2014-09-29 17 43 44](https://cloud.githubusercontent.com/assets/2319504/4440465/917c1b70-47c5-11e4-9314-db5f718486bb.png)
2. _Setup_ the user could we warned by the wizard where appropriate. - ie " for the analysis extent you have chosen, please select higher resolution data or if you want to continue using these data, please zoom out.to a larger analysis extent.
3. _Pre Analysis_ the user should be warned in the form of a popup.
4. Post analysis the metadata about this should be given in a report. Possibly using a bunch of other factors as well each of which could have a ':(' or '!' next to them.
## Overestimation
1. _Training Material_ should inform the user about the overestimation.
2. _Post Analysis_ report should inform the user about this.
## Extents
1. _Training Material_ the user should be informed.
   1._Pre Analysis_ the user should be warned. (with a popup?)
2. _Post Analysis_ the details should be kept in the metadata and this should be added to the report. Again using the ('!' or ':(' notation)
## Appropriate Accuracy
1. _Training_ inform the user that the number given is based on the input data and will most probably not be the 'real' number affected. Need to seek input from users on how they present information and apply then apply similar humanising approaches to summary reports and info graphics ie Less than a Thousand people affected and not the absolute value of 899 etc.
2. _Post Analysis_ keep the population as a whole number. Round depth to closest cm. Calculate rounding after post processors and aggregation.
# RFC Provenance

This RFC was initially proposed at the InaSAFE workshop in Jakarta by:

@Charlotte-Morgan 
@cchristelis
# Record of Votes

@cchristelis +1
# See also:
#1237
#1239

**Comment by timlinux** [comment]
I would still like that we do this and propose moving up the priority to be in an early 4.x release. e.g. 4.1. Many of these items are actually dealt with in standards and notes too but I would also like to see in the dock some contextually aware messaging to the user to try to inform them about possible issues with their planned analysis. @Gustry I am going to make this your baby ok?

**Comment by Charlotte-Morgan** [comment]
+ 1 to make this happen

**Comment by timlinux** [comment]
This would also be good to include as a training module - how to get trustworthy analysis results.
